login-workflow/src/screens/AuthNavigationContainer.tsx

```diff
@@ -72,7 +72,7 @@ export const AuthNavigationContainer: React.FC<NavigationContainerComponentProps
     }, []);
 
     const { routes, routesArray: publicRoutes } = prefixRoutes({ ...defaultRoutes, ...routeConfig });
-    const extraRoutesPathnames = extraRoutes.map((route) => route.props.path);
+    const extraRoutesPathnames = extraRoutes ? extraRoutes.map((route) => route.props.path) : [];
```

Comment:
Oh, I see you noticed this too. I threw up a PR that does this by setting a default value on extraRoutes, but this works too.